packages/instrumentation/src/types.ts

```diff
@@ -0,0 +1,12 @@
+export enum PrismaLayerType {
```

Comment:
This list is not exhaustive and it has some typos in it.
Actually, I'd prefer we remove enum type completely and type new option as the `string`. We have no good way to keep this list up to date without quite a bunch of maintenance burden